src/__internal__/validation-message/__next__/validation-message.style.ts

```diff
@@ -0,0 +1,38 @@
+import styled, { css } from "styled-components";
+
+interface StyledValidationMessageProps {
```

Comment:
suggestion (non-blocking): these won;'t be added to the DOM element so up to you

```
interface StyledValidationMessageProps {
  $isError?: boolean;
  $isDarkBackground?: boolean;
  $isLarge?: boolean;
}